buoy/pkg/git/git_test.go

```diff
@@ -28,30 +30,19 @@ func TestGetRepo_BasicOne(t *testing.T) {
 	repoURL := f.DotGit().Root()
 
 	r, err := GetRepo("foo", repoURL)
-	if err != nil {
-		t.Error("failed to GetRepo: ", err)
-	}
-	if r == nil {
-		t.Error("failed to GetRepo: repo is nil")
-	}
+	require.NoError(t, err)
+	require.NotNil(t, r)
+
 	// TODO: refactor test (use require pkg)
 	// nolint: staticcheck
-	if r.DefaultBranch != "master" {
-		t.Errorf("expected default branch to be master, got %q", r.DefaultBranch)
-	}
-	if want := 2; len(r.Branches) != want {
-		t.Errorf("expected branch count to be %d, got %d", want, len(r.Branches))
-	}
-	if want := 1; len(r.Tags) != want {
-		t.Errorf("expected tag count to be %d, got %d", want, len(r.Branches))
-	}
+	require.Equal(t, r.DefaultBranch, "master")
```

Comment:
Parameterize `master` in the package for when we eventually change the default branch names.

Reply:
@justaugustus the DefaultBranch is filled on the fly, it is not a hardcoded variable 
see here https://github.com/kubernetes-sigs/zeitgeist/blob/master/buoy/pkg/git/git.go#L58-L60

Do you want to make a const for the test? not sure if I follow you